### PR TITLE
[DOC] Starts contribution page

### LINKED ISSUE #618: Add descriptive information for each contributors' contributions to tedana
### Summary
At the last Developers' call we talked about allowing contributors to add descriptive text to explain their involvement with tedana.

### Next Steps

1. Decide where on the site this information will be presented
2. Decide the mechanism through which contributors will add or edit their descriptive texts

### PR DESCRIPTION
**Pull request by jbteves**
Closes #618 .

Changes proposed in this pull request Closes #618 :

- Starts `contributions.md` with a sample sentence and previous funding sources
- Links to README.md under the contributor table, which I suppose we could also move into this file.

Comments welcome, since we didn't have time to discuss much in previous meetings.

### CODE REVIEW

File: contributions.md
Comment (handwerkerd) [inline contributions.md:10]:
```suggestion
- National Institutes of Mental Health, [Section on Functional Imaging Methods](https://fim.nimh.nih.gov) for supporting the 2019 `tedana` hackathon.
```

File: contributions.md
Comment (handwerkerd) [inline contributions.md:11]:
```suggestion
- National Institutes of Health for supporting the [2019 AFNI Code Convergence](https://codeconvergence.org), where work in the 2019 `tedana` hackathon was continued.
```

File: contributions.md
Comment (jbteves) [reply contributions.md:10]: That's true, we're doing all that work on the website, we should link to it!

File: contributions.md
Comment (handwerkerd) [inline contributions.md:6]:
```suggestion
- Dan Handwerker helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
- Joshua Teves helps manage issues and pull requests for a variety of both administrative and code-specific tasks.
```

**Comment by handwerkerd** [comment]
I assume people can add their names & blurbs in alphabetical order?

**Comment by jbteves** [comment]
That was my unstated assumption, yes, @handwerkerd

**Comment by dowdlelt** [comment]
@jbteves Thinking something like this for my sentence:
Logan Dowdle helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data and occasionally adds a new feature (with lots of help!).

File: contributions.md
Comment (handwerkerd) [inline contributions.md:6]:
```suggestion
- Logan Dowdle helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data, and occasionally adds a new feature (with lots of help!).
- Dan Handwerker helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
```

**Comment by eurunuela** [comment]
Thank you @jbteves and @handwerkerd for taking the lead on this. 

Mine could say something like this:

Eneko Uruñuela helps with the development of dimensionality reduction and decomposition algorithms in tedana, with Principal Component Analysis to be more specific, and contributed to the development of the interactive reports.

**Comment by jbteves** [comment]
@eurunuela double check me on 68c85dc

**Comment by eurunuela** [comment]
> @eurunuela double check me on [68c85dc](https://github.com/ME-ICA/tedana/commit/68c85dcb60c592fc5bbc3a1854faef3cb1a641e1)

Looks good, thanks!

File: contributions.md
Comment (tsalo) [inline contributions.md:6]: I think it would be good to either include the person's GitHub username after their full name (e.g., `Logan Dowdle (@dowdlelt)`) or make the name a hyperlink (when desired) to a personal site.

Now that GitHub has personal READMEs, I tend to use my profile for a fairly up-to-date personal description.

**Comment by jbteves** [comment]
Alright, could each of you send me the website you'd like to link to? If you'd like for me to just hyperlink to your GitHub profile, I can do that, too. I'll make it a hyperlink on the bolded name @emdupre if you think that'll work.

File: contributions.md
Comment (jbteves) [reply contributions.md:12]: That's fine. They supported @KirstieJane right?

Comment (emdupre) [reply contributions.md:12]: No, not exactly.. she had a Mozilla fellowship, but that didn't overlap with when she mentored this as a Moz Open Leaders project !

Comment (jbteves) [reply contributions.md:12]: I'm terribly sorry, do you mind writing a short blurb for what they provided? I just don't know because I wasn't participating at that point.

**Comment by mvaziri** [comment]
Thank you @jbteves and @handwerkerd doing this and including me. Mine could say:

Maryam Vaziri-Pashkam helps with the tedana documentation to make it easier to understand for beginners.

This is my website in case you decided to link to those:http://maryam-vaziri.com/

File: contributions.md
Comment (jbteves) [reply contributions.md:6]: With the hyperlink and bolding the names stand out, you can see an example in the rich diff of e99ba95

**Comment by javiergcas** [comment]
For me, it could be:
Javier Gonzalez-Castillo contributed to the development of dimensionality reduction and decomposition algorithms in tedana, as well as to the development of the interactive reports.

File: contributions.md
Comment (jbteves) [reply contributions.md]: I'm happy to link to people's websites if desired, I just need a link if so.

File: contributions.md
Comment (handwerkerd) [reply contributions.md]: More generally: @KirstieJane, We're all adding a sentence or two that self-describes our contributions to tedana. I know your attention is focused elsewhere right now, but I'd like to be able to acknowledge your contributions here. Either make a suggestion to the PR or just put your self-description in a comment & someone else will add it to the text.

**Comment by handwerkerd** [comment]
We can alway add people later, but it would be good to have the steering committee all represented when we merge this. @emdupre & @tsalo, if you have a. chance, could you write a few sentences of self-promotion in the near future?

**Comment by jbteves** [comment]
@tsalo and @emdupre just nudging you to ask if you could contribute a sentence or two about your contributions. I would be really pleased to include this in our fancy new upcoming release!

**Comment by emdupre** [comment]
'Elizabeth DuPre initiated the tedana project in 2017, building on the ME-ICA codebase. She continued to develop the code and began actively creating the community structure as part of the fifth Mozilla Open Leaders cohort (mentored by Kirstie Whitaker). Since her time as interim BDFL, Elizabeth has been involved in most aspects of the project -- although she currently focuses primarily on improving tedana's integration with the broader neuroimaging ecosystem.'

File: contributions.md
Comment (emdupre) [inline contributions.md:18]:
```suggestion
- The Mozilla Open Leaders program for support in developing the tedana community structure as part of [the ME-BIDS project](https://medium.com/read-write-participate/brain-imaging-in-the-open-aac7c17bcf69).
```

File: contributions.md
Comment (tsalo) [inline contributions.md:10]:
```suggestion
- [**Dan Handwerker**][dan-handwerker] helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
- [**Taylor Salo**][taylor-salo] helps with code-related issues and with engaging new contributors to tedana.
- [**Joshua Teves**][joshua-teves] helps manage issues and pull requests for a variety of both administrative and code-specific tasks.
```

File: contributions.md
Comment (tsalo) [inline contributions.md:25]:
```suggestion
[dan-handwerker]: <https://github.com/handwerkerd>
[taylor-salo]: <https://github.com/tsalo>
[joshua-teves]: <https://github.com/jbteves>
```

**Review by tsalo — COMMENTED** [review]
Here's my info.

**Comment by jbteves** [comment]
@tsalo thank you but I think you're being modest!

**Review by tsalo — APPROVED** [review]

**Comment by tsalo** [comment]
Thanks @jbteves! Is there anything else that needs to be done for this PR before it can be merged?

**Comment by jbteves** [comment]
Just need one more approval, that's all.

**Review by eurunuela — APPROVED** [review]
LGTM! I have no comments. Thank you @jbteves

**Pull request merged**
